pkg/deploy/lattice/target_group_manager.go

```diff
@@ -195,22 +197,28 @@ func (s *defaultTargetGroupManager) Delete(ctx context.Context, modelTg *model.T
 	}
 
 	if len(targetsToDeregister) > 0 {
-		deRegisterInput := vpclattice.DeregisterTargetsInput{
-			TargetGroupIdentifier: &modelTg.Status.Id,
-			Targets:               targetsToDeregister,
-		}
+		var deregisterTargetsError error
+		partitions := getPartitionedLatticeTargets(targetsToDeregister, 100)
+		for i, targets := range partitions {
+			deRegisterInput := vpclattice.DeregisterTargetsInput{
```

Comment:
nit: deRegister -> deregister, I know this has been like this before, but lets fix it when we have a chance.

Reply:
+1, same applies to other places like deRegResp -> deregResp